python/spacetime_local/store.py

```diff
@@ -89,6 +89,12 @@ def __flush_derived_objs(self):
             if not tp.__PCC_BASE_TYPE__:
                 self.__objects[tp] = RecursiveDictionary()
 
+    def clear_all(self):
+        self.clear_changes()
+        self.clear_incoming_record()
+        for tp in self.__objects.keys():
+            self.__objects.setdefault(tp, RecursiveDictionary())
```

Comment:
If you are trying to clear all objects from __objects then dont use setdefault.
just do
self.__objects[tp] = RecursiveDictionary()